### PR TITLE
docs: update cloud images for Talos v0.9.3

### PR DESCRIPTION
**Pull request by smira**
Replaces AWS images with the latest 0.9.3, as 0.9.2/0.9.1 contain many
important fixes vs. 0.9.0 release we have today.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: website/src/data/cloud-images.json [hunk 1]
```diff
@@ -489,250 +489,290 @@
   },
   {
     "cloud": "aws",
-    "version": "v0.9.0",
-    "region": "ca-central-1",
+    "version": "v0.9.3",
```
Comment (andrewrynhard) [inline website/src/data/cloud-images.json:492]: Not for this PR, but was thinking that we should put the AMI list into a collapsable section in the docs.

Comment (smira) [reply website/src/data/cloud-images.json:492]: `<details></details>` :+1:

Comment (andrewrynhard) [reply website/src/data/cloud-images.json:492]: That's the one 🙂

**Pull request merged**